Symfony/CS/Console/Command/SelfUpdateCommand.php

```diff
@@ -83,7 +83,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         if ($remoteVersionParsed[0] > $currentVersionParsed[0] && true !== $input->getOption('force')) {
             $output->writeln(sprintf('<info>A new major version of php-cs-fixer is available</info> (<comment>%s</comment>)', $remoteTag));
-            $output->writeln(sprintf('<info>Before upgrading please read</info> https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/%d.%d/UPGRADE.md', $remoteTag[0], $remoteTag[1]));
+            $output->writeln(sprintf('<info>Before upgrading please read</info> https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/%d.%d/UPGRADE.md', $remoteVersionParsed[0], $remoteVersionParsed[1]));
```

Comment:
Actually, please don't assume branch name. Just use tag.
it would look like `https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.0.0-alpha/UPGRADE.md`